app-misc/OpenRGB/OpenRGB-0.6.ebuild

```diff
@@ -12,44 +12,46 @@ else
 	SRC_URI="https://gitlab.com/CalcProgrammer1/OpenRGB/-/archive/release_${PV}/OpenRGB-release_${PV}.tar.bz2"
 	S="${WORKDIR}/OpenRGB-release_${PV}"
 	KEYWORDS="~amd64 ~x86"
+	PATCHES=(
+		"${FILESDIR}"/OpenRGB-0.5-build-system.patch
+	)
 fi
 
 DESCRIPTION="Open source RGB lighting control that doesn't depend on manufacturer software"
 HOMEPAGE="https://gitlab.com/CalcProgrammer1/OpenRGB/"
 LICENSE="GPL-2"
 SLOT="0"
-IUSE="udev"
 
-DEPEND="
+RDEPEND="
 	dev-libs/hidapi:=
-	dev-qt/qtcore:5=
-	dev-qt/qtgui:5=
-	dev-qt/qtwidgets:5=
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
 	virtual/libusb:1
 "
-RDEPEND="${DEPEND}"
+DEPEND="
+	${RDEPEND}
+	dev-cpp/nlohmann_json
+"
 BDEPEND="
 	virtual/pkgconfig
 "
 
+PATCHES+=(
+	"${FILESDIR}"/OpenRGB-0.6-pkgconf.patch
+)
+
 src_prepare() {
 	default
-	rm -rf dependencies/{hidapi,libusb}* || die
-	if [[ ${PV} != *9999* ]]; then
-		eapply "${FILESDIR}/OpenRGB-0.2-build-system.patch"
-	fi
+	rm -r dependencies/{hidapi,libusb,json}* || die
 }
 
 src_configure() {
-	eqmake5
+	eqmake5 INCLUDEPATH+="${ESYSROOT}/usr/include/nlohmann"
 }
 
 src_install() {
 	emake INSTALL_ROOT="${ED}" install
 
 	dodoc README.md OpenRGB.patch
-
-	if use udev; then
-		udev_dorules 60-openrgb.rules
```

Comment:
Why are we not installing the udev rules anymore? AFAICS the rules file is still in the repository, and the upstream instructions seem to suggest they are still needed.

Reply:
It's installed by "make install" already